src/components/settings/AIConfigPanel.tsx

```
@@ -14,6 +14,7 @@ import { buildFinalApiUrl } from '../../utils/apiUrlBuilder';
 import { SliderInput } from '../ui/SliderInput';
 import { useDialog } from '../../hooks/useDialog';
 import { isToolCallCapableApiType } from '../../constants/aiCapabilities';
+import { AIService } from '../../services/aiService';
```

ESLint

[error] '../../services/aiService' import is restricted from being used by a pattern. View components must not import business services directly. Move the call into a hook in src/features/.../hooks/** and import the hook instead. See docs/adr/0001-frontend-layering.md.

(no-restricted-imports)